router/sleeve.go

```diff
@@ -363,6 +370,8 @@ func (sleeve *SleeveOverlay) MakeForwarder(params ForwarderParams) (OverlayForwa
 		specialChan:      specialChan,
 		confirmedChan:    confirmedChan,
 		finishedChan:     finishedChan,
+		establishedChan:  make(chan struct{}),
+		errorChan:        make(chan error, 1),
```

Comment:
The difference is due to a very different pattern of usage from the other channels.  Which fundamentally is because the other channels are an internal implementation detail, for communication with an internal goroutine, whereas these two are part of the OverlayForwarder interface.